department-of-veterans-affairs/caseflow
# Queue |  Prevent clearing all issue dispositions when editing an issue

Resolves #5402 

### Description
After creating/updating an issue, we overwrite the staged appeal's issues with the server's version (i.e. to get the correct server-assigned ID after creation). However, because we haven't updated dispositions yet, we'll end up overwriting dispositions set during the checkout flow.

### Acceptance Criteria 
- [ ] Dispositions set before creating/editing an issue are preserved

### Testing Plan
1. Go to Draft Decision checkout flow, set issue dispositions
2. Create or edit an issue, confirm return to Select Dispositions view
3. Confirm previously-set dispositions are still set

![dispositions_preserved](https://user-images.githubusercontent.com/8533004/39760834-6c4285c2-52a4-11e8-8583-36684a284f8c.gif)
